src/structs/dedup.rs

```diff
@@ -1,3 +1,4 @@
+use std::mem;
```

Comment:
This needs to be `core::hint`

Reply:
Thanks for pointing that out.

I didn't realize that `pushgen` is designed to work with `no_std`

Reply:
Just pushed the new commits